test/integration/targets/aws_secret/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/aws
+unsupported
+unstable
```

Comment:
New tests should not be marked `unstable`. Also, unless we are unable to add permissions to CI to support this test, it should not be marked `unsupported`.

Reply:
@mattclay Is it preferred to keep tests in a failed state until that determination is made? I ran into this with #36200 as well

Reply:
@slapula Yes, that way we can add the necessary permissions to our CI system and simply re-run the tests against the PR. In the event we're unable to add the permissions we will then comment on the PR so the `unsupported` alias can be added.

Reply:
Understood!  I have made the appropriate changes.